patches/draft-js-import-element+1.4.0.patch

```diff
@@ -0,0 +1,26 @@
+diff --git a/node_modules/draft-js-import-element/esm/stateFromElement.js b/node_modules/draft-js-import-element/esm/stateFromElement.js
+index eccabfd..9ec7496 100644
+--- a/node_modules/draft-js-import-element/esm/stateFromElement.js
++++ b/node_modules/draft-js-import-element/esm/stateFromElement.js
+@@ -373,7 +373,7 @@ function () {
+         switch (customInline.type) {
+           case 'STYLE':
+             {
+-              style = style.add(customInline.style);
++             [].concat(customInline.style).forEach(customStyle => { style = style.add(customStyle); });
+               break;
+             }
+ 
+diff --git a/node_modules/draft-js-import-element/lib/stateFromElement.js b/node_modules/draft-js-import-element/lib/stateFromElement.js
```

Comment:
This file doesn't actually need fixing on my setup, but don't know if that's just a local thing. Fixed both exported JS files to be safe. Ping @swissspidy.